examples/semi_supervised/plot_label_propagation_structure.py

```diff
@@ -61,7 +61,7 @@
     marker=".",
     label="unlabeled",
 )
-plt.legend(scatterpoints=1, shadow=False, loc="upper right")
+plt.legend(scatterpoints=1, shadow=False, loc="center")
 plt.title("Raw data (2 classes=outer and inner)")
```

Comment:
Let's also avoid printing the output of the matplotlib title while we are at it.

```suggestion
_ = plt.title("Raw data (2 classes=outer and inner)")
```